lib/p2p/Peer.ts

```diff
@@ -190,6 +192,21 @@ class Peer extends EventEmitter {
 
     // TODO: Check that the peer's version is compatible with ours
 
+    // request peer's known nodes only if p2p.discover option is true
+    if (this.discover) {
+      if (this.discoverminutes === 0) {
+        // timer is disabled
+        this.sendPacket(new packets.GetNodesPacket());
+        this.discoverTimer = undefined; // defensive programming
+      } else {
+        // timer is enabled
+        this.sendPacket(new packets.GetNodesPacket()); // initial execution
+        this.discoverTimer = setInterval(() => {
```

Comment:
perhaps follow the `pingTimer` one-liner, by creating a non-anonymous method `sendGetNodes`.

Reply:
sure